### DISCUSSION TITLE
Can't update oh-my-zsh

### DISCUSSION THREAD

**Discussion by albertogfm** [ohmyzsh/ohmyzsh#9906] [Help] [answered]
So I was trying to update oh-my-zsh but when y typed Y it showed me the next message:
![image](https://user-images.githubusercontent.com/42499609/117898580-e1e45280-b28a-11eb-9a25-92214399f95f.png)
any thougts?

**Comment by mcornella** [comment, marked as answer]
Your local copy of the repository got broken because of a missing/corrupt object. The easiest solution is to clone the repository again, unless you made local modifications to it. If that's the case, you'll have to download the missing object again from a healthy repository and overwrite the missing object with the downloaded one.
